client/src/app/app.component.ts

```diff
@@ -99,10 +105,14 @@ export class AppComponent implements OnInit {
     this.projectService
       .getAllProjects()
       .pipe(
-        catchError(() => {
+        catchError((response: HttpErrorResponse) => {
+          if (response.error.status === 401) {
```

Comment:
When is the interceptor jumping in? Isn't it intercepting even before this line is executed?

Reply:
Right, I was a bit on the wrong track due to a bug in the interceptor which is fixed now. In 401 cases the interceptor needs to return an `EMPTY` observable in order to let the app component render the login page through the router outlet in this case.